### ISSUE TITLE
`borderline south` in a breakable box leads to "missing character" messages in the log

### ISSUE THREAD

**Issue by u-fischer** [T-F-S/tcolorbox#33]
In breakable boxes the arguments of `borderline south` (and probably also `borderline north`) are left in the input stream when the line is not printed and then lead to `Missing character` messages in the log:

```
\documentclass{article}
\usepackage[many]{tcolorbox}

\newtcolorbox{mybox}{
 breakable,
 enhanced,
 borderline south={2mm}{0pt}{gray},
 }
 
% suggested change:
%\makeatletter
%\def\tcb@bordersouth#1#2#3{%
%  \ifdefstring{\tcb@split@state}{unbroken}{\tcb@bordersouth@draw{#1}{#2}{#3}}{%
%    \ifdefstring{\tcb@split@state}{last}{\tcb@bordersouth@draw{#1}{#2}{#3}}{}}%
%}

\usepackage{lipsum}
\begin{document}
\begin{mybox}
abc\lipsum

\end{mybox}

\end{document}
``` 

gives

```
Missing character: There is no 2 in font nullfont!
Missing character: There is no m in font nullfont!
Missing character: There is no m in font nullfont!
Missing character: There is no 0 in font nullfont!
Missing character: There is no p in font nullfont!
Missing character: There is no t in font nullfont!
Missing character: There is no g in font nullfont!
Missing character: There is no r in font nullfont!
Missing character: There is no a in font nullfont!
Missing character: There is no y in font nullfont!
```

Imho it would be better if `\tcb@bordersouth`  would grab the arguments and throw them away if not needed.

**Comment by T-F-S** [comment]
Yes, this is a bug. I will correct this for the next version of tcolorbox using your suggestion or something similar which consumes the arguments.

Thank you for reporting this bug.

**Comment by T-F-S** [comment]
Solved with version 4.12.
